draft-packages/form/KaizenDraft/Form/Primitives/Input/styles.scss

```diff
@@ -192,73 +203,114 @@ $input-disabled-border-alpha: 50%;
 // Default
 .input.default {
   background-color: $color-white;
-  color: $dt-color-form-text-color;
-  border-color: $dt-color-form-border-color;
+  color: $color-purple-800;
   display: flex;
   align-items: center;
 
   @include form-input-placeholder {
-    color: $dt-color-form-text-color;
+    color: $color-purple-800;
     opacity: $input-placeholder-opacity;
   }
 
-  &:focus:not([disabled]),
-  &:hover:focus:not([disabled]),
-  &:hover:not([disabled]) {
-    border-color: $dt-color-form-border-color-hover;
+  @include form-input-focus-state {
+    background-color: $color-gray-200;
   }
 
   &:focus + .focusRing {
-    border-color: $dt-color-form-border-color-focus;
+    border-color: $color-blue-500;
   }
 
   &.disabled {
+    background-color: $color-white;
+    border-color: rgba($color-gray-500-rgb, $input-disabled-opacity);
     color: rgba($color-purple-800-rgb, $input-disabled-opacity);
 
     @include form-input-placeholder {
       opacity: $input-disabled-opacity;
     }
   }
 
-  &.error:not(:focus) {
+  &:not(.error):not(.caution) {
+    border-color: $color-gray-500;
+
+    &:disabled {
+      border-color: rgba($color-gray-500-rgb, $input-disabled-opacity);
+    }
+
+    @include form-input-focus-state {
+      border-color: $color-gray-600;
+    }
+  }
+
+  &.error {
     border-color: $color-red-500;
+
+    &.disabled {
+      border-color: rgba($color-red-500-rgb, $input-disabled-opacity);
+    }
+  }
+
+  &.caution {
+    border-color: $color-yellow-600;
+
+    &.disabled {
+      border-color: rgba($color-yellow-600-rgb, $input-disabled-opacity);
+    }
   }
 }
 
 // Reversed (Dark Backgrounds)
 .input.reversed {
+  background: transparent;
   color: $color-white;
-  border-color: rgba($color-white-rgb, 0.65);
 
   @include form-input-placeholder {
     color: $color-white;
     opacity: $input-placeholder-opacity;
   }
 
-  &:focus:not([disabled]),
-  &:hover:not([disabled]) {
+  @include form-input-focus-state {
     background: rgba($color-white-rgb, 0.1);
-    border-color: $color-white;
   }
 
   &:focus + .focusRing {
-    border-color: $dt-color-form-border-color-reversed-focus;
+    border-color: $color-blue-300;
+  }
+
+  &:not(.error):not(.caution) {
+    border-color: rgba($color-white-rgb, 0.65);
+
+    &:disabled {
+      border-color: rgba($color-white-rgb, $input-disabled-opacity);
+    }
```

Comment:
I see this is a common pattern. You could create a mixin to make this cleaner :)

Reply:
I've updated the Input primitive to extent InputHTMLAttributes and then used that in the Text Field. As discussed yesterday I've left in the inputValue and defaultInputValue props to allow for a gentle deprecation of these

Reply:
After talking about how opacity for disabled I found some docs that talk about how disabled states [should work](https://cultureamp.design/guidelines/interaction-states/#disabled-state) - I'm starting to think we could split this into its own ticket and just gut the specific targeting of inner elements and just apply it to the component wrapper